Samples/WinMLSamplesGallery/WinMLSamplesGallery/shaders.hlsl

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+struct PSInput
+{
```

Comment:
what is this shader for?

Reply:
It's used in the pipeline state object of the d3d12 sample. It was in the native project before but I moved it here since this is where files are searched for in the published version of the gallery